src/assets/tests/digitech.json

```diff
@@ -0,0 +1,623 @@
+        "text": "Dreierkompliment",
+        "value": false
+      },
+      {
+        "text": "Einerkomplement",
+        "value": true
+      },
+      {
+        "text": "Zweierkomplement",
+        "value": true
+      }
+    ]
+  },
+  {
+    "name": "Wie wird eine (Dezimal-) Kommazahl als Festkommazahl dargestellt/ermittelt?",
+    "type": "dropList",
+    "orientation": "vertical",
+    "answer": [
+      {
+        "text": "π ⋅ 2ª, wenn a = Nachkommastellen",
+        "value": true
+      },
+      {
+        "text": "round(Zahl) (runden einer Zahl zur nächsten ganzen Zahl)",
+        "value": true
+      },
+      {
+        "text": "Konvertieren ins Binärsystem",
+        "value": true
+      },
+      {
+        "text": "Verschiebe Zahl um a Bit nach rechts.(((Komma nach links)))",
+        "value": true
+      }
+    ]
+  },
+  {
+    "name": "Mit welcher Wortbreite ist laut IEEE 754r eine Gleitkommazahl vom Typ Double definiert?",
+    "type": "checkbox",
+    "answer": [
+      {
+        "text": "8 Bit",
+        "value": false
+      },
+      {
+        "text": "24 Bit",
+        "value": false
+      },
+      {
+        "text": "64 Bit",
+        "value": true
+      },
+      {
+        "text": "256 Bit",
+        "value": false
+      }
+    ]
+  },
+  {
+    "name": "Welche Klassen von Primimplikanten gibt es?",
+    "type": "checkbox",
+    "answer": [
+      {
+        "text": "Kernprimimplikant",
+        "value": true
+      },
+      {
+        "text": "Abzählbarer Primimplikant",
+        "value": false
+      },
+      {
+        "text": "Absolut eliminierbarer Primimplikant",
+        "value": true
+      },
+      {
+        "text": "definitiv eliminierbarer Primimplikant",
+        "value": false
+      },
+      {
+        "text": "relativ eliminierbarer Primimplikant",
+        "value": true
+      },
+      {
+        "text": "unabhängiger Primimplikant",
+        "value": false
+      }
+    ]
+  },
+  {
+    "name": "Wozu nutzt man Primimplikanten?",
+    "type": "checkbox",
+    "answer": [
+      {
+        "text": "Vereinfachung eines Terms.",
+        "value": true
+      },
+      {
+        "text": "Zur Minimierung eines Terms",
+        "value": true
+      },
+      {
+        "text": "Erstellen eines Terms mit maximaler Ordnung",
+        "value": true
+      }
+    ]
+  },
+  {
+    "name": "KV-Diagramme nutzt man zur...",
+    "type": "checkbox",
+    "answer": [
+      {
+        "text": "... Beschäftigung bei Langeweile.",
+        "value": false
+      },
+      {
+        "text": "... graphischen Vereinfachung eines Terms.",
+        "value": true
+      },
+      {
+        "text": "... Darstellung des Minterms/Maxterms.",
+        "value": true
+      },
+      {
+        "text": "... Erstellung einer Wahrheitstabelle.",
+        "value": true
+      }
+    ]
+  },
+  {
+    "name": "In welchen Größen darf man Primimplikanten ermitteln? (x^n bedeutet: x hoch n)",
+    "type": "radio",
```

Comment:
this can't be a radio type, maybe checkbox?

Reply:
my bad, only thought about x^n. is checkbox now.